devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy

```diff
@@ -104,7 +104,16 @@ def installed() {
 	if (!childDevices) {
 		addChildButtons(numberOfButtons)
 	}
-	sendEvent(name: "supportedButtonValues", value: ["pushed"])
+    if(childDevices) {
+		def event
+		for(def endpoint : 1..device.currentValue("numberOfButtons")) {
+			event = createEvent(name: "button", value: "pushed", isStateChange: true)
```

Comment:
Add `displayed: false` to this event so that it doesn't show up in event history

Reply:
Remember to make this change too